src/Hedgehog/Hedgehog.fsproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.6;netstandard2.0;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
I don't recommend this.  You are reintroducing bug #199 here.

Reply:
#242 is a recent discuss about which versions to target.

Reply:
I can add these back in. I think we should try to target the net standard versions unless that's impossible. The impetus for this PR was that I couldn't build the code in a Linux environment at all, which is why there were build failures on previous PRs.

Has this been mostly developed on Windows thus far?

Reply:
Which SDK did you try?  I think the recently released .NET 5.0.100 SDK or the previous .NET Core 3.1.404 SDK should be able to build this code base on Linux including all the targets.  I could be wrong though.

Reply:
@TysonMN This is the output I get for the dotnet version. I'm using a docker container to develop in, so I only have core.

```
$ dotnet --version
3.1.403
```

I tried building with various target frameworks but for one reason or another I wasn't able to get a compile/test run to work. Threading the needle was not possible for me.

If I'm understanding correctly, this is a deployment concern, not a development one? As in, developers of Hedgehog are more or less running with .NET Core, but we have to target older frameworks for NuGet packaging?

Reply:
Huh.  I thought that should work.

This is work is similar to the work needed to complete issue #215.  Want to help with that?

Reply:
Following the guidance [here][1], I can add `netstandard1.6` back in, anything lower than that fails to build.

[1]: https://docs.microsoft.com/en-us/dotnet/standard/net-standard#which-net-standard-version-to-target

Reply:
> This is work is similar to the work needed to complete issue #215. Want to help with that?

As part of this PR, you mean?

Reply:
For simplicity, let's have a different PR for that work

Reply:
Working on that other issue could help us reach the same understanding about building on Linux because we can have the GitHub Action execute on a Linux machine, possibility see it fail, and look at the logs to see what didn't work.